Trent edits 20230927 (#82)
jessegmeyerlab/proteomics-tutorial: content/07.peptide-protein-labeling.md

## Peptide Quantification {.page_break_before}
<!-- todo: change the label of this file eventually to peptide quant instead of peptide labeling
 -->

### Label-free quantification (LFQ) of peptides
LFQ of peptide precursors requires no additional steps in the protein extraction, digestion, and peptide purification workflow. 
Samples can be taken straight to the mass spectrometer and are injected one at a time, each sample necessitating their own LC-MS/MS experiment and raw file. 

> Quantification of peptides by LFQ is routinely performed by many commercial and freely available proteomics software. 

Reviewer: ```suggestion
Quantification of peptides by LFQ is routinely performed by many commercial and freely available proteomics software (see Data Analysis section below). 
```

In LFQ, peptide abundances across LC-MS/MS experiments are calculated by aligning windows of accurate peptide mass and retention time.

### Stable isotope labeling of peptides
One approach to improve the throughput and quantitative completeness within a group of samples is sample multiplexing via stable isotope labeling.
Multiplexing enables pooling of samples and parallel LC-MS/MS analysis within one run.
Quantification can be achieved at the MS1- or MSn-level, dictated by the upstream labeling strategy.

> Metabolic labeling, e.g. stable isotope labeling in amino acid cell culture (SILAC) or , and chemical labeling, e.g. amine-modifying tags for relative and absolute quantification (mTRAQ) or dimethyl labeling, provide MS1-level peptide quantification. 

Reviewer: ```suggestion
Stable isotope labeling methods produce peptides that are chemically identical from each sample that differ only in their mass due. 
Methods include stable isotope labeling in amino acid cell culture (SILAC) [@DOI:10.1074/mcp.m200025-mcp200] and chemical labeling such as amine-modifying tags for relative and absolute quantification (mTRAQ) [@DOI:10.1074/mcp.M111.014423] or dimethyl labeling [@PMID:19300442].
```

The labeling of each sample imparts mass shifts (e.g. 4 Da, 8 Da) which can be detected within the MS1 full scan.

> The ability to label samples in cell culture enables impactful quantitative biology experiments [@DOI:https://doi.org/10.1016/j.celrep.2013.01.003; https://doi.org/10.1038/nmeth.1647].

Reviewer: ```suggestion
The ability to label samples in cell culture has enabled impactful quantitative biology experiments [@DOI:10.1016/j.celrep.2013.01.003; @DOI:10.1038/nmeth.1647].
```

These approaches have nearly exclusively been performed using data-dependent acquisition (DDA) strategies. 

> However, recent work employing faster instrumentation has shown the benefits of chemical labeling with 3-plex mTRAQ labels for data-independent acquisition (DIA) [@DOI: https://doi.org/10.1038/s41587-022-01389-w; https://doi.org/10.15252/msb.202211503. 

Reviewer: ```suggestion
However, recent work employing faster instrumentation has shown the benefits of chemical labeling with 3-plex mTRAQ or dimethyl labels for data-independent acquisition (DIA) [@DOI:10.1038/s41587-022-01389-w; @DOI:10.15252/msb.202211503], an idea originally developed nearly a decade earlier using chemical labels to quantify lysine acetylation and succinylation stoichiometry [@DOI:10.1007/s13361-016-1476-z]. 
```

> As new tags with higher plexing become available, strategies like plexDIA and mDIA are sure to benefit [@DOI: https://doi.org/10.1038/s41587-022-01389-w; https://doi.org/10.15252/msb.202211503].

Reviewer: ```suggestion
As new tags with higher plexing become available, strategies like plexDIA and mDIA are sure to benefit [@DOI:10.1038/s41587-022-01389-w; @DOI:10.15252/msb.202211503].
```

### Peptide labeling with isobaric tags
Another approach to improve throughput and quantitative completeness within a group of samples is multiplexing via isobaric labels, a strategy which enables parallel data acquisition after pooling of samples. 

> Commercial isobaric tags include tandem mass tags (TMT) [@DOI: https://doi.org/10.1021/ac0262560] and isobaric tags for relative and absolute quantification (iTRAQ) [@DOI: https://doi.org/10.1074/mcp.M400129-MCP200] amongst others, and several non-commercial options have also been developed [@DOI: https://doi.org/10.1021/ac902778d]. 

Reviewer: ```suggestion
Commercial isobaric tags include tandem mass tags (TMT) [@DOI:10.1021/ac0262560] and isobaric tags for relative and absolute quantification (iTRAQ) [@DOI:10.1074/mcp.M400129-MCP200] amongst others, and several non-commercial options have also been developed [@DOI:10.1021/ac902778d]. 
```

> 10- and 11-plex TMT kits were recently supplanted by proline-based TMT tags (TMTpro), originally introduced as 16-plex kits in 2019 [@DOI:https://doi.org/10.1021/acs.analchem.9b04474] and upgraded to an 18-plex platform in 2021 [@DOI:10.1021/acs.jproteome.1c00168]. 

Reviewer: ```suggestion
10- and 11-plex TMT kits were recently supplanted by proline-based TMT tags (TMTpro), originally introduced as 16-plex kits in 2019 [@DOI:10.1021/acs.analchem.9b04474] and upgraded to an 18-plex platform in 2021 [@DOI:10.1021/acs.jproteome.1c00168]. 
```

The isobaric tag labeling-based peptide quantitation strategy uses derivatization of every peptide sample with a different isotopic incorporation from a set of isobaric mass tags. 
All isobaric tags have a common structural theme consisting of 1) an amine-reactive groups (usually triazine ester or N-hydroxysuccinimide [NHS] esters) which react with peptide N-termini and ε-amino group of the lysine side chain of peptides, 2) a balancer group, and 3) a reporter ion group.  
Peptide labeling is followed by pooling the labelled samples, which undergo MS and MS/MS analysis simultaneously. 
As the isobaric tags are used, peptides labeled with these tags give a single MS peak with the same precursor m/z value in an MS1 scan and identical retention time of liquid chromatography analysis. 
The modified parent ions undergo fragmentation during MS/MS analysis generating two kinds of fragment ions: (a) reporter ions and (b) peptide fragment ions. 
Each reporter ions’ relative intensity is directly proportional to the peptide abundance in each of the starting samples that were pooled. 

> As with peptide LFQ, the b- and y-type fragment ion peaks are still used to identify amino acid sequences of peptides, from which proteins can be inferred. 

Reviewer: ```suggestion
As usual, the b- and y-type fragment ion peaks are still used to identify amino acid sequences of peptides, from which proteins can be inferred. 
```

> Since it is possible to label most tryptic peptides with an isobaric mass tag at least at the n-termini, numerous peptides from the same protein can be detected and quantified, thus leading to an increase in the confidence in both protein identification and quantification [@PMID:25337643, https://doi.org/10.1146/annurev-biodatasci-080917-013516]. 

Reviewer: ```suggestion
Since it is possible to label most tryptic peptides with an isobaric mass tag at least at the n-termini, numerous peptides from the same protein can be detected and quantified, thus leading to an increase in the confidence in both protein identification and quantification [@PMID:25337643, @DOI:10.1146/annurev-biodatasci-080917-013516]. 
```

#### Tandem Mass Tag (TMT)
TMT labeling is based on a similar principle as that of iTRAQ. 
In the case of 6-plex-TMT, the masses of reporter groups are roughly 126, 127, 128, 129, 130, and 131 Thompsons [@PMID:26584918]. 

> TMT works best with MS which allow quantitation at MS3-level with higher accuracy (e.g.: Thermo’s Orbitrap Tribrid instruments) [@PMID:25337643; https://doi.org/10.1021/acs.analchem.3c01155] by limiting the ratio distortion of co-isolated precursors within isobaric multiplexed quantitative proteomics [@PMID:21963607; https://doi.org/10.1021/ac502040v].

Reviewer: ```suggestion
TMT works best with MS which allow quantitation at MS3-level with higher accuracy (e.g.: Thermo’s Orbitrap Tribrid instruments) [@PMID:25337643; @DOI:10.1021/acs.analchem.3c01155] by limiting the ratio distortion of co-isolated precursors within isobaric multiplexed quantitative proteomics [@PMID:21963607; @DOI:10.1021/ac502040v].
```

> Infrared photoactivation of co-isolated TMT fragment ions generates more quantitative reporter ion generation and sensitivity relative to standard beam-type collisional activation [@DOI:https://doi.org/10.1021/acs.analchem.1c05398] 

Reviewer: ```suggestion
Infrared photoactivation of co-isolated TMT fragment ions generates more quantitative reporter ion generation and sensitivity relative to standard beam-type collisional activation [@DOI:10.1021/acs.analchem.1c05398] 
```

TMT is widely used for quantitative protein biomarker discovery. 
In addition, TMT labeling technique helps multiplex sample analysis enabling efficient use of instrument time. 
TMT labelling also controls for technical variation because after samples are mixed the ratios are locked in, and any sample loss would be equal across channels. 

> A wide range of TMT reagents with different multiplexing capabilities are available, such as TMT zero, TMT duplex, TMT six plex, TMT 10-plex, and TMT 11-plex are available along with the recent addition of TMT 16-plex [@DOI:https://doi.org/10.1021/acs.analchem.9b04474] and now TMT 18-plex [@DOI:10.1021/acs.jproteome.1c00168]. 

Reviewer: ```suggestion
A wide range of TMT reagents with different multiplexing capabilities are available, such as TMT zero, TMT duplex, TMT six plex, TMT 10-plex, and TMT 11-plex are available along with the recent addition of TMT 16-plex [@DOI:10.1021/acs.analchem.9b04474] and now TMT 18-plex [@DOI:10.1021/acs.jproteome.1c00168]. 
```
